src/utils/saga/index.js

```diff
@@ -0,0 +1,11 @@
+import { counter, pager, paged } from './paged';
+import { errorToast, successToast, toast } from './toasts';
+
+export const helpers = {
```

Comment:
This could be

```js
import pageUtilities from './paged';
import toastUtilities from './toasts';

export default {
  ...pageUtilities,
  ...toastUtilities
}
```

Then you can still `import helpers from 'utils/saga';` or do a "destructured" import `import { pager } from 'utils/saga';` - but you can't destructure in an import so you'd never be able to do something like `import { helpers: { pager } } from 'utils/saga';` with the current implementation.

Reply:
Updated